src/components/expanded-state/swap-settings/SwapSettingsState.js

```diff
@@ -137,10 +138,12 @@ export default function SwapSettingsState({ asset }) {
             <Text align="center" color="primary" size="18px" weight="bold">
               {lang.t('exchange.settings')}
             </Text>
-            <SourcePicker
-              currentSource={currentSource}
-              onSelect={updateSource}
-            />
+            {network !== Network.arbitrum && (
```

Comment:
hm this hides the picker, maybe it's better to remove the icon and block the press context menu inside `SourcePicker`?

Reply:
@g-a-v-i-n approves this so nvm